### DISCUSSION TITLE
PowerToys v0.67.1 Translation v17.3

### DISCUSSION THREAD

**Discussion by zetaloop** [zetaloop/PowerToys-CN#17] [Announcements]
**PowerToys v0.67.1**
Translation v17.3

Update.

_Translated source code incremental pack.
Compressed using 7-Zip._

<hr /><em>This discussion was created from the release <a href='https://github.com/ZetaSp/PowerToys-Chinese-TransMOD/releases/tag/v17.3'>PowerToys v0.67.1 Translation v17.3</a>.</em>

**Comment by AlonWangg** [comment]
蓝奏云盘官方的第二个压缩包下载不了

**Reply by zetaloop** [reply]
可以从[https://github.com/microsoft/PowerToys/releases/](https://github.com/microsoft/PowerToys/releases/)下载官方安装包，这里只是备用

**Reply by zetaloop** [reply]
可能是蓝奏云屏蔽了分块压缩包，但是我没钱充vip（
